src/dynamic-pages/collections/sections/month-rank.tsx

```diff
@@ -49,18 +49,23 @@ const HeaderCell = styled(NumberCell)(() => ({
 export default withInViewContainer(function MonthRankSection() {
   const { collection } = useContext(CollectionsContext);
 
-  const { dimension, tabs } = useDimensionTabs('monthly-rankings', true);
-  const asyncData = useCollectionMonthRank(collection?.id, dimension.key);
+  const { dimension, tabs, dateType } = useDimensionTabs('monthly-rankings', true);
+  const asyncData = useCollectionMonthRank(collection?.id, dimension.key, dateType);
 
   return (
     <section>
       <H2 id="monthly-rankings">Month-to-month Ranking</H2>
-      <P2>The following table ranks repositories using three metrics: stars, pull requests, and issues. The table compares the most recent two months of data and indicates whether repositories are moving up or down the rankings.</P2>
+      <P2>
+        The following table ranks repositories using three metrics: stars, pull
+        requests, and issues. The table compares the most recent two months of
+        data and indicates whether repositories are moving up or down the
+        rankings.
```

Comment:
The following table ranks repositories using three metrics: stars, pull requests, and issues. The table compares last 28 days or the most recent two months of data and indicates whether repositories are moving up or down the rankings.